style.css

```diff
@@ -369,4 +369,78 @@ body {
     justify-content: space-between;
     column-gap: 1.6rem;
   }
+
+  .contact-me {
+    background-image: url("./assets/images/contact-form-shapes-mobile.png");
+    background-color: #6070ff;
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding: 10.4rem  2.4rem 0.8rem 2.4rem;
+    margin-top: 17.685rem;
+    border-top-left-radius: 6rem;
+  }
```

Comment:
- [ ] Kindly stick with the *Figma* design as closely as possible, it seems like your design is not matching in your work. Please have a look at the below comparison table to understand the problem in a better way through visualization. Fortunately, I got the issue and the issue is a typo, your folder name is `Assets` not **assets**, please fix this typo and at the end of this comment you can see the suggestion.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-06 22-29-06](https://github.com/AjroudiRida/Portfolio/assets/66476329/c4033e38-d12d-4026-aa52-ff75cc24ff43)


</td>

<td>

![Screenshot from 2023-06-06 21-01-24](https://github.com/JoyTaribagshaw/My-Portfolio/assets/66476329/4241651c-5ad9-4ec3-b29b-e48c9624dd52)


</td>
</tr>
</table>


#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)

```suggestion
  .contact-me {
    background-image: url("./assets/images/contact-form-shapes-mobile.png");
    background-color: #6070ff;
    background-size: contain;
    background-position: right;
    background-repeat: no-repeat;
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
    padding: 10.4rem  2.4rem 0.8rem 2.4rem;
    margin-top: 17.685rem;
    border-top-left-radius: 6rem;
  }
```


### Missing Decorative Image

![image](https://github.com/AjroudiRida/Portfolio/assets/66476329/96eaad98-e5cf-4393-966a-0815a900732d)